### PR TITLE
[optimize] Only display member records whose approver information is not empty

### LINKED ISSUE #177: KCC 成员登记权限控制
image
KCC 成员信息登记按钮直接跳转到飞书表单页，能否加权限控制，登记了由站长审核后才在官网显示

### LINKED ISSUE #178: KCC 社区信息登记审核
与 #177 同理，自发组建其他城市的 KCC 也应该可以审核

### PR DESCRIPTION
**Pull request by huoyueyuyuan**
Checklist（清单）:

- [x] Labels
- [ ] Assignees
- [x] Reviewers

- Closes #177
- Closes #178

### CODE REVIEW

File: pages/community/[community].tsx [hunk 1]
```diff
@@ -21,16 +21,12 @@ export const getServerSideProps = compose<
...
-  async ({ params: { community } = {} }) => {
-    const list = await new CommunityMemberModel().getAll({ community });
-
-    return { props: JSON.parse(JSON.stringify({ list, community })) };
-  },
-);
+>(errorLogger, translator(i18n), async ({ params: { community } = {} }) => {
+  const list = (await new CommunityMemberModel().getAll({ community })).filter(
+    member => !!member.approver,
+  );
```
Comment (TechQuery) [inline pages/community/[community].tsx:27]:
```suggestion
  const list = await new CommunityMemberModel().getAll({ community });
```
在下方加配置即可，从后端接口查询的源头来过滤掉：

https://github.com/kaiyuanshe/kaiyuanshe.github.io/blob/20ce808f9a6c1b38e1954bf02dc0dde55a1ed786/models/Community/CommunityMember.ts#L38-L38

**Review by TechQuery — CHANGES_REQUESTED** [review]
1. 社区本身也应该有申请审核机制
2. 所有表单对应按钮的文案也要体现是“申请”

File: pages/community/[community].tsx [hunk 2]
```diff
@@ -22,9 +22,7 @@ export const getServerSideProps = compose<
     community: string;
   }
 >(errorLogger, translator(i18n), async ({ params: { community } = {} }) => {
-  const list = (await new CommunityMemberModel().getAll({ community })).filter(
-    member => !!member.approver,
-  );
+  const list = await new CommunityMemberModel().getAll({ community });
   return { props: JSON.parse(JSON.stringify({ list, community })) };
```
Comment (TechQuery) [inline pages/community/[community].tsx]:
```suggestion
  const list = await new CommunityMemberModel().getAll({ community });

  return { props: JSON.parse(JSON.stringify({ list, community })) };
```

File: translation/zh-CN.ts [hunk 3]
```diff
@@ -217,7 +217,7 @@ export default {
   community_member: '社区成员',
   add_member: '数据整理中，敬请期待',
   community: '社区',
-  member_register: 'KCC 成员信息登记',
+  member_register: '申请成为 KCC 成员',
   activity_register: 'KCC 活动信息登记',
-  community_register: '社区信息登记',
+  community_register: 'KCC社区创建申请',
```
Comment (TechQuery) [inline translation/zh-CN.ts:222]:
```suggestion
  community_register: 'KCC 社区创建申请',
```

**Review by TechQuery — CHANGES_REQUESTED** [review]

File: translation/en-US.ts [hunk 4]
```diff
@@ -222,9 +222,9 @@ export default {
 
   //community detail page
   community_member: 'Community Members',
-  add_member: 'Community member data is being compiled, please stay tuned',
+  add_member: 'Community Member Data Is Being Compiled, Please Stay Tuned',
```
Comment (PapayaHUANG) [inline translation/en-US.ts:225]:
```suggestion
  add_member: 'Community Member Data is being Compiled, Please Stay Tuned',
```
非实义词不用首字母大写

File: translation/en-US.ts [hunk 5]
```diff
@@ -222,9 +222,9 @@ export default {
...
   community: ' Community',
-  member_register: 'KCC Member Information Registration',
+  member_register: 'Apply To Become A KCC Member',
```
Comment (PapayaHUANG) [inline translation/en-US.ts:227]:
```suggestion
  member_register: 'Apply to Become a KCC Member',
```

File: translation/en-US.ts [hunk 6]
```diff
@@ -222,9 +222,9 @@ export default {
...
   activity_register: 'KCC Activity Information Registration',
-  community_register: 'Community Information Registration',
+  community_register: 'KCC Community Creation Application',
```
Comment (PapayaHUANG) [inline translation/en-US.ts:229]:
```suggestion
  community_register: 'KCC Community Establishment Application',
```

**Review by PapayaHUANG — CHANGES_REQUESTED** [review]

**Review by TechQuery — APPROVED** [review]

**Review by PapayaHUANG — APPROVED** [review]

**Pull request merged**